### PR TITLE
Swissue153 - GPIO Telemetry Bug

### PR DESCRIPTION
**Pull request by kb1lqd**
This firmware update corresponds to software pull request: https://github.com/FaradayRF/Faraday-Software/pull/166

This PR also addresses firmware issue: https://github.com/FaradayRF/Faraday-Firmware/issues/72

Changes:

* Removed unneeded telemetry variable in Telemetry.c
* Fixed telemetry GPIO frame creation to pack GPIO bits into frame with correct bit-shifts
* Fixed GPIO allocation header file to properly allocate definitions to digital out pins
* Update version per firmware commit

### CODE REVIEW

File: Applications/Telemetry/Telemetry.c [hunk 1]
```diff
@@ -212,26 +212,34 @@ void application_telem_create_rf_pkt(unsigned char *packet, char *src_callsign,
...
+		telem_packet_3_struct.gpio_state |= (P3OUT & GPIO_2) << 2;// GPIO 2
+		telem_packet_3_struct.gpio_state |= (P4OUT & GPIO_3) >> 1;// GPIO 3
+		telem_packet_3_struct.gpio_state |= (P4OUT & GPIO_4) << 1;// GPIO 4
+		telem_packet_3_struct.gpio_state |= (P4OUT & GPIO_5) << 3;// GPIO 5
+		telem_packet_3_struct.gpio_state |= (P4OUT & GPIO_6) << 5;// GPIO 6
+		telem_packet_3_struct.gpio_state |= (P4OUT & GPIO_7) << 7;// GPIO 7
+
+		telem_packet_3_struct.io_state	= 0x00;
+		telem_packet_3_struct.io_state 	|= (P5IN & ARDUINO_IO_8) << 5;		// Grab IO 8 and shift left 5 to put into BIT 8 of IO_State
+		telem_packet_3_struct.io_state 	|= (P5IN & ARDUINO_IO_9) << 3;		// Grab IO 9 and shift left 3 to put into BIT 7 of IO_State
```
Comment (kb1lqc) [inline Applications/Telemetry/Telemetry.c]: IO9 doesn't exist anymore

**Review by kb1lqc — DISMISSED** [review]
All looks good except we no longer even have an Arduino IO 9 channel and we really shouldn't call any channels "Arduino"

**Comment by kb1lqd** [comment]
@kb1lqc Changes implemented please review.

**Review by kb1lqc — APPROVED** [review]
Looks good, make sure to document that this part of telemetry will probably be random even if we don't really use it anymore. That way 1) we know what to expect 2)we know we have space to send more info down

**Pull request merged**